### ISSUE TITLE
Investigate update of Patternfly to be compatible with OS Console

### ISSUE THREAD

**Issue by jshaughn** [kiali/kiali#4836]
Epic: https://github.com/kiali/kiali/issues/4260

**Comment by jshaughn** [comment]
As expected, there is some negative impact wrt popper/tippy.  This is a tricky area because cytoscape requires cytsoscape-popper, which in turn requires popper.js, which uses tippy.  And we also use popper.js independently.  So getting a combination that is happy is not easy.  Some observations:
- updating cytoscape from 3.15.5 to 3.16+ can negatively impact our label sizes, I see this for dagre on initial render of a smaller graph.
- popper.js 1 is deprecated, and is superseded by  v2 at @popperjs/core
- #patternfly/react-core depends on tippy.js, which depends on popper.js
- react-datepicker also depends on popper.js
- the latest cytoscape-popper depends on v2 @popperjs/core

So, it seems we can't update to popper v2 and we have rendering issues with cy 3.16+.  This limits our ability to move forward with those dependencies.  But, we seem to have some popper/tippy issues afther the pf-react updates, I'm not sure why but I guess it's more related to the pf changes than popper/tippy, since that didn't really change.

**Comment by lucasponce** [comment]
> the latest "cytoscape-popper" depends on v2 @popperjs/core

Mmm, if this is blocking, we could review and clone this to have the freedom to control this (and future clashes).
I looked at the code and it's less complex that the layouts.

So, I don't see a "why not", it's a tradeoff but it seems it can simplify this conflict.

**Comment by jshaughn** [comment]
>  we could review and clone this 

I don't think this will really enable us to move forward because pf-react itself is not using popper v2.  Until that moves forward I think we are bound to v1.  Yesterday I tried to move to v2 and failed, today I will see if I can resolve the issues I see just maintaining v1.  From what I can see PF deprecated some popper/tippy stuff AND disabled it, so not sure if that's our issue, but will investigate.

**Comment by nrfox** [comment]
Can we use the [resolutions](https://classic.yarnpkg.com/lang/en/docs/selective-version-resolutions/) field to try and make each dependency happy?

**Comment by jshaughn** [comment]
> Can we use the resolutions field to try and make each dependency happy?

As it stands right now, I don't think we want to try and incorporate popper v2.  I think it makes more sense to match pf-react.
